app/customer.py

```diff
@@ -0,0 +1,93 @@
+from dataclasses import dataclass
+
+import datetime
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int
+    car: object = Car
+
+    def current_balance(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def calculate_distance(self,
+                           shop_detail: Shop) -> float:
+        distance = ((sqrt((shop_detail.location[0]
+                           - self.location[0]) ** 2
+                          + (shop_detail.location[1]
+                             - self.location[1]) ** 2)))
+        return distance
+
+    def calculate_gasoline(self,
+                           fuel_price: int,
+                           shop_detail: Shop) -> int:
```

Comment:
I’m not sure it returns int, I believe it returns float

Reply:
Done